### DISCUSSION TITLE
atomic_dipole.npy dataset

### DISCUSSION THREAD

**Discussion by 919607** [deepmodeling/deepmd-kit#2165] [Q&A] [answered]
How do I get the atomic_dipole.npy dataset before I train a deep Wannier model for Wannier centroids? Currently my initial dataset was calculated statically by vasp, with five files: coord.npy, energy.npy, force.npy, virial.npy and box.npy, but no atomic_dipole.npy file.

**Comment by wanghan-iapcm** [comment, marked as answer]
You need to compute the Wannier centers and provide the Wannier centroid as the averge of Wannier centers.

**Reply by 919607** [reply]
Thank you very much for your reply. 
Can I use dpdata to generate the required .npy file if I have calculated Wannier centers.xyz with wannier90?

**Reply by wanghan-iapcm** [reply]
by far dpdata does not support extracting wannier centroid from the wannier90 outputs. You may prepare the data file by yourself.

**Reply by 919607** [reply]
Ok, I will try. Thanks for your help

**Reply by Sbandyo1** [reply]
Could you please explain how to compute the wannier centres? (I know how it  using wannier90. )

Since, you did not use wannier90 for the original dataset, how you manage to do that?

Thanks,
